src/main/java/im/status/keycard/SecureChannel.java

```diff
@@ -176,6 +199,66 @@ public void mutuallyAuthenticate(APDU apdu) {
     respond(apdu, len, ISO7816.SW_NO_ERROR);
   }
 
+  /**
+   * Processes the LOAD_CERTS command. Copies the APDU buffer into `certs`.
+   * This function expects a DER signature and may only be called once.
+   * @param apdu the JCRE-owned APDU object.
+   */
+  public void loadCert(APDU apdu) {
+    byte[] apduBuffer = apdu.getBuffer();
+
+    final short caPubOffset = (short) ISO7816.OFFSET_CDATA;
+    final short certOffset = (short) (caPubOffset + PUBKEY_LEN);
+    final short certVerifyLen = (short)(CERTIFICATE_LEN - ECDSA_MAX_LEN);
+
+    if (idCertStatus != ID_CERTIFICATE_EMPTY) {
+      // Card cert may only be set once and never overwritten
+      ISOException.throwIt(ISO7816.SW_COMMAND_NOT_ALLOWED);
+    }
+
+    // Make sure the received certificate is appropriate length
+    if (apduBuffer[ISO7816.OFFSET_LC] != (byte) (PUBKEY_LEN + CERTIFICATE_LEN)) {
+      ISOException.throwIt(ISO7816.SW_DATA_INVALID);
+    }
+
+    JCSystem.beginTransaction();
+    // Save the certificate
+    Util.arrayCopy(apduBuffer, certOffset, idCertificate, (short) 0, CERTIFICATE_LEN);
```

Comment:
You aren't saving the pubkey

- [ ] Save pubkey in addition to cert

Reply:
The CA pubkey or the card pubkey? Card pubkey is held locally in `idKeyPair`. The CA pubkey is not needed, so why would I save it?

Reply:
To clarify: last week's discussion where we decided that the card should not validate a Lattice certificate, means the card doesn't need the CA pubKey

Reply:
Right, forgot about that change! We don't need the cert signer's pubkey if we aren't validating Lattice certs. My bad